Sources_v3/StreamChatUI/ChatChannel/Reactions/ChatMessageReactionsView.swift

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageReactionsView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
```

Comment:
Loud thoughts:

Aint it too much view classes for just small reaction bubble?
At this point I found myself on thought: "Wait, I have seen this one, haven't i?"

Reply:
Actually, we have the same count + 2 for bubbles (base and default one). It can feel we've got too many because previously all (an a VC) were located in the single file.